exercises/poker/.meta/src/reference/java/Hand.java

```diff
@@ -2,108 +2,129 @@
 import java.util.stream.Collectors;
 
 class Hand {
-    private String input;
-    private int score;
+	private String input;
+	private int score;
 
-    Hand(String hand) {
-        this.input = hand;
-        this.score = scoreHand(parseCards(hand));
-    }
+	Hand(String hand) {
+		this.input = hand;
+		this.score = scoreHand(parseCards(hand));
+	}
 
-    int getScore() {
-        return score;
-    }
+	int getScore() {
+		return score;
+	}
 
-    String getInput() {
-        return input;
-    }
+	String getInput() {
+		return input;
+	}
 
-    private List<Card> parseCards(String hand) {
-        ArrayList<Card> parsedCards = new ArrayList<>();
-        String[] cardsToParse = hand.split(" ");
-        for (String cardToParse : cardsToParse) {
-            parsedCards.add(new Card(cardToParse));
-        }
-        return parsedCards;
-    }
+	private List<Card> parseCards(String hand) {
+		ArrayList<Card> parsedCards = new ArrayList<>();
+		String[] cardsToParse = hand.split(" ");
+		for (String cardToParse : cardsToParse) {
+			parsedCards.add(new Card(cardToParse));
+		}
+		return parsedCards;
+	}
 
-    private Map<Integer, Integer> getFrequencyMap(List<Card> cards) {
-        Map<Integer, Integer> frequencyMap = new HashMap<>();
-        for (Card c : cards) {
-            if (frequencyMap.containsKey(c.getRank())) {
-                frequencyMap.put(c.getRank(), frequencyMap.get(c.getRank()) + 1);
-            } else {
-                frequencyMap.put(c.getRank(), 1);
-            }
-        }
-        return frequencyMap;
-    }
+	private Map<Integer, Integer> getFrequencyMap(List<Card> cards) {
+		Map<Integer, Integer> frequencyMap = new HashMap<>();
+		for (Card c : cards) {
+			if (frequencyMap.containsKey(c.getRank())) {
+				frequencyMap.put(c.getRank(), frequencyMap.get(c.getRank()) + 1);
+			} else {
+				frequencyMap.put(c.getRank(), 1);
+			}
+		}
+		return frequencyMap;
+	}
 
-    private int scoreHand(List<Card> cards) {
-        List<Card> cardsByRank = cards
-                .stream()
-                .sorted(Comparator.comparing(Card::getRank))
-                .unordered()
-                .collect(Collectors.toList());
+	private int scoreHand(List<Card> cards) {
+		List<Card> cardsByRank = cards.stream().sorted(Comparator.comparing(Card::getRank)).unordered()
```

Comment:
It's considered good practice to add a linebreak with each full stop here, as it was before. That makes the code easier to read and maintain 🙂

The same goes for all the later changes you've made 🙂